libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -258,6 +258,9 @@ const AP_Param::GroupInfo AP_GPS::var_info[] = {
 // constructor
 AP_GPS::AP_GPS()
 {
+    static_assert((sizeof(_initialisation_blob) * 9) < (4800 * GPS_BAUD_TIME_MS * 1e-3),
```

Comment:
... Yes totally forgot that one :p

I'll swap to CHAR_BIT, but if it's ever not 8 bits to byte I suspect you will suddenly see a bunch of anomalies in the GPS data :)